src/libraries/System.Formats.Cbor/tests/Reader/CborReaderTests.cs

```diff
@@ -54,6 +54,79 @@ public static void Depth_ShouldReturnExpectedValue(int depth)
             Assert.Equal(CborReaderState.Finished, reader.PeekState());
         }
 
+        [Fact]
+        public static void Reset_ConformanceMode()
```

Comment:
I guess it depends on the use case, but it seems odd to me that a reader instance can be reset to a new conformance mode. That being said, it would require a different `Reset` overload accepting a conformance mode parameter.

Reply:
I made a typo in that statement, I meant to say "cannot be reset" :-)

I'm suggesting that you should be able to reset a reader to follow a new conformance mode, e.g. by doing something like `reader.Reset(CborConformanceMode.Ctap2Canonical);`. That's dependent on a new overload being approved though, so it's not blocking this PR.

Reply:
Resetting the conformance mode was something "available" early in the issue that led to Reset, but in the end it was scoped down to just accepting new data.